model/minsky.cc

```diff
@@ -295,23 +293,57 @@ namespace minsky
                              }
                          }
                      return false;
-                   });
+                   });               
+    
+    // curved wires not always included in the new group, add them if they aren't. For ticket 1190.
+    if (canvas.selection.numWires()!=g->numWires())
+      {
+        canvas.selection.recursiveDo(&GroupItems::wires,
+                       [&](Wires&, Wires::iterator w) {
+                         auto alreadyAdded = g->findAny
+                           (&GroupItems::wires,
+                            [&w](const WirePtr& u)
+                            {return u.get()==w->get();});
+                                      
+                         if (!alreadyAdded) {  
+                           auto f=w->get()->from(), t=w->get()->to();
+                           g->addWire(new Wire(f,t,w->get()->coords()));
```

Comment:
I have already given a reason why wires may not be pasted (potential double definition being the main one), so I don't think this a good idea to do this blindly here. We need to establish why wires are not being added, and fix any issue there.